src/lib/gossip_net/gossip_net.ml

```diff
@@ -181,8 +183,9 @@ module Make (Message : Message_intf) :
                        mismatch due to forgery
                      *)
                     Linear_pipe.force_write_maybe_drop_head
```

Comment:
then you can use `Strict_pipe.Writer.write` here, and wrap the whole thing in `to_linear_pipe`.